### PR TITLE
Added integer support for Explore

### LINKED ISSUE #1561: Int labels cause crash in Evaluation.explore()
Repro steps:
```python
import turicreate as tc
train = tc.SFrame({'images': tc.SArray([[0,0,0,0,0,0,0,0,0],[1,1,1,1,1,1,1,1,1]]).pixel_array_to_image(3, 3, 1), 'label': [0, 1]})
test = train
model = tc.image_classifier.create(train, target='label')
evaluation = model.evaluate(test)
evaluation.explore()
```

Expected: no crashing

Actual:
```
Unhandled exception in thread started by <function _start_process at 0x11a8f37d0>
Traceback (most recent call last):
  File "/Users/zach/venv/lib/python2.7/site-packages/turicreate/toolkits/image_classifier/_evaluation.py", line 179, in _start_process
    _process_value(value, extended_sframe, proc, evaluation)
  File "/Users/zach/venv/lib/python2.7/site-packages/turicreate/toolkits/image_classifier/_evaluation.py", line 167, in _process_value
    proc.stdin.write(str(_json.dumps(__get_incorrects(json_value['label'], extended_sframe, evaluation)))+"\n")
  File "/Users/zach/venv/lib/python2.7/site-packages/turicreate/toolkits/image_classifier/_evaluation.py", line 128, in __get_incorrects
    filtered_sframe = sf.filter_by([label], "target_label")
  File "/Users/zach/venv/lib/python2.7/site-packages/turicreate/data_structures/sframe.py", line 4416, in filter_by
    column_name + "' in SFrame.")
TypeError: Type of given values does not match type of column 'target_label' in SFrame.
```

### PR DESCRIPTION
**Pull request by abhishekpratapa**
fixes #1561

### CODE REVIEW

File: src/unity/python/turicreate/visualization/_plot.py [hunk 1]
```diff
@@ -34,7 +34,7 @@ def _focus_client_app():
             delay .5
             tell application \"Turi Create Visualization\" to activate
             '''
-    focus = __subprocess._Popen(['osascript', '-'], stdout=_PIPE, stdin=_PIPE)
+    focus = _Popen(['osascript', '-'], stdout=_PIPE, stdin=_PIPE)
```
Comment (znation) [inline src/unity/python/turicreate/visualization/_plot.py]: This fix seems unrelated... are you building on a branch with multiple commits?

Comment (abhishekpratapa) [reply src/unity/python/turicreate/visualization/_plot.py]: Oh no, this was another bug I discovered in the focusing mechanism. It was a fix I made to my branch and never committed. It accidentally got included in this PR.

Comment (znation) [reply src/unity/python/turicreate/visualization/_plot.py]: Accidentally or intentionally? Do you intend for this change to go in with this PR, or should it be factored out into another PR?

Comment (abhishekpratapa) [reply src/unity/python/turicreate/visualization/_plot.py]: I accidentally added it. I'll refactor it out into another PR.

Comment (abhishekpratapa) [reply src/unity/python/turicreate/visualization/_plot.py]: Updated PR!
